src/arch/common/special.sh

```diff
@@ -6,7 +6,7 @@ export CHARMINC=../include
 if test -n "$BUILD_CUDA"
 then
   echo "---------------------- special.sh for cuda executing ----------------"
-  if [ "$CMK_GDIR" == "gni" ] ; then
+  if [ "$CMK_GDIR" = "gni" ] ; then
```

Comment:
Shouldn't this stay as double equal signs?

Reply:
Double equal signs are only valid in bash’s “[[“ operator, not in “[“.